fhd_core/gridding/visibility_degrid.pro

```diff
@@ -229,10 +229,10 @@ FOR bi=0L,n_bin_use-1 DO BEGIN
           box_matrix, vis_n, beam_int=beam_int, beam2_int=beam2_int, n_grp_use=n_grp_use,degrid_flag=1,_Extra=extra)
     endif else begin
         IF interp_flag THEN $
-          FOR ii=0L,vis_n-1 DO box_matrix[psf_dim3*ii]=$
+          FOR ii=0L,vis_n-1 DO box_matrix[double(psf_dim3)*ii]=$
```

Comment:
Instead of casting `psf_dim3` to a double every time it is used, we should change it's type on line 147 where it is defined. Also, could it be a Long64 instead of a double? If that works, it would be preferable.